sopel/cli/run.py

```diff
@@ -346,9 +346,10 @@ def get_configuration(options):
     except config.ConfigurationNotFound as error:
         print(
             "Welcome to Sopel!\n"
-            "I can't seem to find the configuration file, "
-            "so let's generate it!\n")
-        settings = utils.wizard(error.filename)
+            "I can't seem to find the configuration file.\n"
+            "To start the configuration wizard, run `sopel configure`"
+        )
+        raise error
```

Comment:
What would you say to printing the string representation of `error` instead of "I can't seem to find the configuration file."? Then the error cause and the name of the missing/expected file would be in one place, and Sopel wouldn't repeat itself.

Reply:
Done:
```
$ sopel -c baz
Welcome to Sopel!
Unable to find the configuration file /home/user/.sopel/baz
To start the configuration wizard, run `sopel configure`
$
```